### PR TITLE
READY: Query and get result rows as tuples

### PR DESCRIPTION
**Pull request by lukebakken**
Includes and supercedes #1390 

Convert output lists to tuples for TTB; fix get notfound

### CODE REVIEW

**Comment by hmmr** [comment]
Can you do this for `get`? The change needs to be here:https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_ts_svc.erl#L493

**Comment by lukebakken** [comment]
@hmmr it's already in this branch & PR:

https://github.com/basho/riak_kv/blob/fixes/ensure-result-rows-as-tuples/src/riak_kv_ts_svc.erl#L493

File: src/riak_kv_qry_worker.erl [hunk 1]
```diff
@@ -292,9 +292,10 @@ prepare_final_results2(#riak_sel_clause_v1{col_return_types = ColTypes,
 
 prepare_final_results_test() ->
     Rows = [[12, <<"windy">>], [13, <<"windy">>]],
```
Comment (javajolt) [inline src/riak_kv_qry_worker.erl]: Are you using `Rows` any more?  If not, the build will probably fail.

**Comment by hazen** [comment]
👍 32ffb5a

**Pull request merged**